src/user_utils.jl

```diff
@@ -57,53 +112,15 @@ function demo(output_file::String,
               camera_position::Tuple{Float32, Float32, Float32},
               camera_orientation::Tuple{Float32, Float32, Float32},
               screen_distance::Float32,
-              renderer_type::Type{<:Renderer},
+              renderer_type::String,
```

Comment:
I really do not like that this function argument is a string instead of a `Renderer` subtype. Using a string works well when using a CLI but it becomes cumbersome and inefficient when the code is used in the REPL or in a script. Furthermore, if we use subtypes of `Renderer` if the user makes a typo it will have a compiling error from Julia when calling the function instead of a runtime error when cheching if a `String` is valid.

Reply:
Addressed in [this commit](https://github.com/Paolo97Gll/Raytracer.jl/pull/20/commits/afc627f6f70661936287be44ff40fa415fd1a7d2)